tools/crm_resource.c

```diff
@@ -1053,202 +1055,6 @@ validate_cmdline_config(void)
     }
 }
 
-/*!
- * \internal
- * \brief Get the <tt>enum pe_find</tt> flags for a given command
- *
- * \return <tt>enum pe_find</tt> flag group appropriate for \c options.rsc_cmd.
- */
-static uint32_t
```

Comment:
We'll need to be extra careful in reviewing to make sure these old functions got translated to correct flags for all commands.